tests/test_dicom_eager.py

```diff
@@ -13,14 +13,32 @@
 # the License.
 # ==============================================================================
 """Tests for DICOM."""
+
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
 
+import os
+import pytest
+
 import tensorflow as tf
+import tensorflow_io.dicom as dicom_io  # pylint: disable=wrong-import-position
+
+
 if not (hasattr(tf, "version") and tf.version.VERSION.startswith("2.")):
   tf.compat.v1.enable_eager_execution()
-import tensorflow_io.dicom as dicom_io  # pylint: disable=wrong-import-position
+
```

Comment:
@marcelolerendegui The line of enable_eager_execution() ideally should be placed immediately after `import tensorflow as tf`:
```
import tensorflow as tf
if not (hasattr(tf, "version") and tf.version.VERSION.startswith("2.")):
   tf.compat.v1.enable_eager_execution()
import tensorflow_io.dicom as dicom_io  # pylint: disable=wrong-import-position
```

The reason is that TensorFlow can only work in either eager mode or non eager mode. However, this mode has to be done immediately after `import tensorflow as tf` as otherwise there will be unintended consequences.